src/main.c

```diff
@@ -1706,6 +1712,7 @@ static void cmdline_handler(int argc, char **argv)
 			"  checksum\tUpdate a manifest with checksums (and optionally sign it)\n" \
 			"  install\tInstall a bundle\n" \
 			"  info\t\tShow file information\n" \
+			SERVICE_HELP_TEXT \
```

Comment:
As this is used only in one place, why use the define above (instead of just `#if` / `#endif`)?

Reply:
I had some troubles with the multi-line string and #if in between. If you have a working solution, I would agree that this is preferable ;)

Reply:
Drop the trailing backslashes and it will work.

Reply:
@a3f You're right. I thought it should work and tested it, but obviously did something wrong or drew the wrong conclusions.. I've now removed them all and inserted the define in-place. Looks much cleaner now :+1: